app/scripts/constants/sentry-state.ts

```diff
@@ -270,6 +270,11 @@ export const SENTRY_BACKGROUND_STATE = {
     useTransactionSimulations: true,
     enableMV3TimestampSave: true,
   },
+  FeatureFlagController: {},
```

Comment:
I'm guessing this is leftover from when the controller had a different name?

```suggestion
```

Reply:
Good catch ! 2818e50fa6515143dc8de161211db8aed49e3154